src/backend/common/DefaultMemoryManager.cpp

```diff
@@ -342,12 +349,9 @@ void DefaultMemoryManager::userUnlock(const void *ptr) {
 bool DefaultMemoryManager::isUserLocked(const void *ptr) {
     memory_info &current = this->getCurrentMemoryInfo();
     lock_guard_t lock(this->memory_mutex);
-    locked_iter iter = current.locked_map.find(const_cast<void *>(ptr));
-    if (iter != current.locked_map.end()) {
-        return iter->second.user_lock;
-    } else {
-        return false;
-    }
+    auto iter = current.locked_map.find(const_cast<void *>(ptr));
```

Comment:
nit: I am aware this variable is not used in a very large scope, but it could also use a better name to help while reading the code.